pkg/v1/tkg/aws/client.go

```diff
@@ -307,6 +307,15 @@ func ensureTanzuMissionControlPermissionsForRole(statements []iamv1.StatementEnt
 		Effect: iamv1.EffectAllow,
 		Action: iamv1.Actions{
 			"ec2:DescribeKeyPairs",
+			"ec2:DescribeInstanceTypeOfferings",
+			"ec2:DescribeInstanceTypes",
+			"ec2:DescribeAvailabiilityZones",
```

Comment:
```suggestion
			"ec2:DescribeAvailabilityZones",
```

Reply:
Also, there is a typo below, can you fix it as well:

			"iam:ListAttachedRolePoliices",

Reply:
Thanks for catching this. Updating them.